File: __tests__/__fixtures__/output/java/asynchttp/cookies.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("GET", "https://httpbin.org/cookies")
  .setHeader("cookie", "foo=bar; bar=baz")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();

File: src/targets/php/guzzle/fixtures/cookies.php
<?php
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('GET', 'https://httpbin.org/cookies', [
  'headers' => [
    'cookie' => 'foo=bar; bar=baz',
  ],
]);

echo $response->getBody();